src/types.ts

```diff
@@ -52,15 +76,22 @@ export type RegisterPaymentProps = {
   [x: string]: any
 }
 
-export type TransactionResponse = {
+export type TransactionBaseResponse = {
   id: string
   installationId: string
-  deviceId: string
   riskAssessment: RiskAssessment
   reasons: Array<Reason>
+}
+
+export type TransactionResponse = TransactionBaseResponse & {
+  deviceId: string
```

Comment:
This should be on `TransactionBaseResponse`. Web transactions also have the `deviceId` in the response.

Reply:
How so?

![image](https://github.com/inloco/incognia-node/assets/1139664/123c797a-6ec1-447d-a436-eff39ae9929c)

Is the API Reference outdated?

Reply:
Unfortunately, it is :(
It is described on the docs page: https://developer.incognia.com/docs/us/v6/apis/login-api#web-login-risk